prepare

```diff
@@ -41,28 +41,6 @@ pull_snap() {
         print_error "Could not set up Snap! git submodule.\nPlease make sure that you have access to the Internet and git is installed in your system."
         exit 1
     fi
-    cd src/snap
-    git checkout master
-    if ! git pull; then
-        print_error "Could not pull latest Snap! version from Git.\nPlease make sure that you have access to the Internet and git is installed in your system."
-        cd ../..
-        exit 1
-    fi
```

Comment:
Yes, but two different things...
  - In this provisional state, latest Snap! is older than our version (with the pending PR includes). It have not sense to pull latest master. Really it is a delicate situation... and maybe the final changes in Snap are different and not compatible with the chosen PRs.
  - In a "normal" state, I think a "normal" prepare ("--all" or "--snap") must take the Snap! snapshot pointed (and tested) in our repo. Developers can test everything (they can do a "git pull" or we can build another prepare option) but the latest Snap! version could not be the better choice. Sometimes, Snap! could go faster than Snap4Arduino, and sometimes we must fix 'decorators' and other things to upgrade snap version.